src/database/mod.rs

```diff
@@ -510,3 +511,40 @@ pub struct RepoModel {
     pub tree_state: TreeState,
     pub created_at: DateTime<Utc>,
 }
+
+/// Represents a comment that will be minimized later.
+pub struct CommentModel {
+    pub id: PrimaryKey,
+    /// The GitHub repository this comment belongs to.
+    pub repository: GithubRepoName,
+    /// The pull request number this comment is associated with.
+    pub pr_number: PullRequestNumber,
+    /// The label of the comment.
+    pub label: CommentLabel,
+    /// The node id of the comment in GitHub.
+    pub node_id: String,
+    pub created_at: DateTime<Utc>,
+}
+
+impl sqlx::Type<sqlx::Postgres> for CommentLabel {
+    fn type_info() -> sqlx::postgres::PgTypeInfo {
+        <String as sqlx::Type<sqlx::Postgres>>::type_info()
+    }
+}
+
+impl sqlx::Encode<'_, sqlx::Postgres> for CommentLabel {
+    fn encode_by_ref(
+        &self,
+        buf: &mut sqlx::postgres::PgArgumentBuffer,
+    ) -> Result<sqlx::encode::IsNull, BoxDynError> {
+        <String as sqlx::Encode<sqlx::Postgres>>::encode(self.to_string(), buf)
```

Comment:
Let's implement the DB (de)serialization here, no need to tie it with `FromStr`/`ToString`. I know that some existing code in bors already does it, but it's not a great design.